<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
    <line>
      <![CDATA[Test Case Failed:Fail]]>
    </line>
    <line>
      <![CDATA[TestCase Pass: Pass]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="13014" started-at="2017-06-02T05:04:46Z" finished-at="2017-06-02T05:04:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13014" started-at="2017-06-02T05:04:46Z" finished-at="2017-06-02T05:04:59Z">
      <class name="test.Demo">
        <test-method status="PASS" signature="print1()[pri:1, instance:test.Demo@598067a5]" name="print1" duration-ms="48" started-at="2017-06-02T10:34:46Z" finished-at="2017-06-02T10:34:46Z">
          <reporter-output>
            <line>
              <![CDATA[Test Case Failed:Fail]]>
            </line>
          </reporter-output>
        </test-method> <!-- print1 -->
        <test-method status="FAIL" signature="print2()[pri:2, instance:test.Demo@598067a5]" name="print2" duration-ms="12841" started-at="2017-06-02T10:34:46Z" finished-at="2017-06-02T10:34:59Z">
          <exception class="junit.framework.ComparisonFailure">
            <message>
              <![CDATA[expected:<Goog[le]> but was:<Goog[]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.ComparisonFailure: expected:<Goog[le]> but was:<Goog[]>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.Assert.assertEquals(Assert.java:107)
	at test.Demo.print2(Demo.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- print2 -->
        <test-method status="PASS" signature="print3()[pri:3, instance:test.Demo@598067a5]" name="print3" duration-ms="5" started-at="2017-06-02T10:34:59Z" finished-at="2017-06-02T10:34:59Z">
          <reporter-output>
            <line>
              <![CDATA[TestCase Pass: Pass]]>
            </line>
          </reporter-output>
        </test-method> <!-- print3 -->
      </class> <!-- test.Demo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
